airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -471,17 +474,19 @@ def wait_for_job(self, url: str) -> str:
                     if not error_message:
                         # not all failed response can have "errorMessage" and we need to show full response body
                         error_message = job_info
-                    self.logger.error(f"JobStatus: {job_status}, sobject options: {self.sobject_options}, error message: '{error_message}'")
-
+                    self.logger.error(
+                        f"Job: {self.name}/{job_id}, JobStatus: {job_status}, sobject options: {self.sobject_options}, error message: '{error_message}'"
+                    )
+                else:
+                    self.logger.info(f"Job: {self.name}/{job_id}, JobStatus: {job_status}, sobject options: {self.sobject_options}")
```

Comment:
In case of success, do we need to print the `self.sobject_options`?

Reply:
removed